src/specs/models/processing-config-bodies/ProcessingConfigConfigureEmbedding.v1.yaml

```diff
@@ -0,0 +1,51 @@
+title: Configure Embedding
+type: object
+properties:
+  embeddingSettings:
+    type: object
+    required:
+      - method
+      - methodSettings
+    properties:
+      method:
+        type: string
+        minLength: 1
+      methodSettings:
+        type: object
+        properties:
+          umap:
+            type: object
+            properties:
+              minimumDistance:
+                type: number
+              distanceMetric:
+                type: string
+                minLength: 1
+          tsne:
+            type: object
+            properties:
+              perplexity:
+                type: number
+              learningRate:
+                type: number
+  clusteringSettings:
+    type: object
+    required:
+      - method
+      - methodSettings
+    properties:
+      method:
+        type: string
+        minLength: 1
+      methodSettings:
+        type: object
+        properties:
+          louvain:
+            type: object
+            properties:
+              resolution:
+                type: number
+required:
+  - embeddingSettings
+  - clusteringSettings
```

Comment:
in the UI, the embedding settings are treated as a different entity from clustering settings -- the user sends requests for embedding settings change when clicking the Apply button, whereas for the clustering settings, these get changed after the user moves the slider (where the actual work behind this is not implemented in the backend). Because we will never really send embedding and clustering settings in one request, I don't think that it makes sense to have them like that tied to one schema and also making them both required.

Reply:
I see, this is how it was designed originally, I just refactored into separate files. We can always change the schema if we need to.